pandas/tests/series/test_constructors.py

```diff
@@ -651,6 +656,17 @@ def test_constructor_dict(self):
         expected.iloc[1] = 1
         assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize('input_class', [list, tuple])
```

Comment:
I believe we also accept a duck-typed list class that is not a generator here? (IOW an iterable).